fedmsg_meta_fedora_infrastructure/tests/github.py

```diff
@@ -1834,6 +1834,112 @@ class TestGithubRelease(Base):
         }
     }
 
+class TestGithubPageBuild(Base):
+    """ There exists `a service
+    <https://apps.fedoraproject.org/github2fedmsg>`_ to link the select github
+    repos of fedora contributors with the fedmsg bus.
+
+    Messages of *this* type are published whenever someone **builds a Github Page**.
+    """
+    expected_title = "github.page_build"
+    expected_subti = None
```

Comment:
And for the expected subtitle here, can you make it something like ``"relrod rebuilt the github.io page for fedora-infra/fas3-api-haskell"``?